tests/jobs/csv_export2/test_integration.py

```diff
@@ -186,3 +189,37 @@ def verifyExportedFile(exported_file_name, result_header, result_iter_rel):
         set(dframe[["ENSEMBLE", "REAL"]].itertuples(index=False, name=None))
         == result_iter_rel
     )
+
+
```

Comment:
Should we have `@pytest.mark.integration` here (requires declaring it in `setup.cfg`)?

Reply:
That does make sense, we should still default to running all the tests though, as this is a plugin, `ert` will always be available.